docs/jobs.rst

```diff
@@ -0,0 +1,147 @@
+.. _jobs:
+
+Jobs
+####
+
+A script will aggregate build information from Mozilla archives, and another is in charge of keeping it up to date.
+
+.. note::
+
+    The ``user:pass`` in the command-line examples is the Basic auth for Kinto.
+
+.. image:: overview.png
+
+Currently we use `Kinto <http://kinto-storage.org>`_ as a generic database service. It allows us to leverage its simple API for storing and querying records. It also comes with a set of client libraries for JavaScript, Python etc.
+
+.. note::
+
+    More specific solutions may replace it when the product scope evolves. Like for example fetching information from:
+
+    * `Automatic Update Service (AUS, a.k.a Balrog) <https://wiki.mozilla.org/Balrog>`_
+
+
+Initialization
+==============
+
+We provide an initialization manifest that will define the buckets and collection and their permissions.
+
+Load it with:
+
+.. code-block:: bash
+
+    kinto-wizard load --server https://kinto/ --auth user:pass initialization.yaml
+
+The JSON schema validation can be enabled on the server with the following setting:
+
+.. code-block:: ini
+
+    kinto.experimental_collection_schema_validation = true
+
+
+Build for Amazon Lambda
+=======================
+
+In order to build the AWS Lambda Zip archive in an isolated environment, we use Docker:
+
+* ``make get_zip``
+
+
+Load latest S3 inventory
+========================
+
+A command to download the latest S3 manifests from Mozilla archives, and create records on a remote Kinto server.
```

Comment:
This might be outside the scope of this PR because so far it seems like you're just moving stuff around, but it might be good to add a line or two about what the contents of the S3 manifests are. For example, `A command to download the latest S3 manifests, containing information about all releases available on archive.mozilla.org, and send that information as Buildhub records to a remote Kinto server`. Actually I think the manifest doesn't even contain the information about the releases available, it's just a list of filenames, which we then scrape, right?